src/ml/neural_net/mps_layer_conv_padding.m

```diff
@@ -0,0 +1,65 @@
+#import <ml/neural_net/mps_layer_conv_padding.h>
+
+@implementation TCMPSConvolutionPadding
+
++ (BOOL) supportsSecureCoding {
```

Comment:
Based on my reading [here](https://developer.apple.com/documentation/foundation/nssecurecoding), this should probably return `NO`. It doesn't look like key/value coding is properly implemented in this interface.

Reply:
Or let the superclass `NSObject` handle it?

Reply:
Unfortunately we can't let NSObject handle it. But you're right that we should return `NO`. I've updated the values.